readme-cmake.md

```diff
@@ -75,14 +75,22 @@ gcc-c++ cmake ninja-build SDL2-devel libpng-devel libjpeg-turbo-devel mesa-libGL
 Here's a summary of every command you will need for development:
 
 ```bash
-$ cmake --preset <preset>               # configure project (only needs to be done once)
-$ cmake --build --preset <preset>-debug # actually build Endless Sky (as well as any tests)
-$ ./build/<preset>/Debug/endless-sky    # run the game
-$ ctest --preset <preset>-test          # run the unit tests
-$ ctest --preset <preset>-benchmark     # run the benchmarks
-$ ctest --preset <preset>-integration   # run the integration tests (Linux only)
+$ cmake --preset <preset>                     # configure project (only needs to be done once)
+$ cmake --build --preset <preset>-debug       # actually build Endless Sky (as well as any tests)
+$ ./build/<preset>/Debug/endless-sky          # run the game
+$ ctest --preset <preset>-test                # run the unit tests
+$ ctest --preset <preset>-benchmark           # run the benchmarks
+$ ctest --preset <preset>-integration         # run the integration tests (Linux only)
```

Comment:
And is there a reason why this is still Linux only? If we can run integration tests on all platforms in debug-mode, then I guess we can also run them on all platforms in non-debug-mode.

Reply:
Btw, thanks for making the integration tests more accessible. :)

Reply:
It's currently Linux only because other platforms can't create an offscreen window easily (and on Linux we can use SDL or Xfvb). I can certainly enable them for the other two platforms, but they will constantly create new windows. If that is acceptable, than sure :smile:

Reply:
Good point, let's leave this as-is for now. (I have seen the constant new window creation during some earlier testing, let's not give that to our users.)